_documentation/user/flags/colorblindness.md

```diff
@@ -5,4 +5,18 @@ type: Bad
 summary: Can't tell team colors. Don't shoot teammates!
 ---
 
-...
+The Colorblindness flag causes a player to see all players as rogue players and identify no longer identifies players by name for them.
+
+## Tactics
+
+Colorblindness causes teammates and enemies to look exactly alike. The risk of shooting a teammate increases dramatically and enemies can easily blend in with teammates.
+
+Countering the effects of Colorblindness requires good communications with teammates to know their positions and in order to locate enemies quickly.
+The main difficulties are that positions may quickly become outdated and also stealth may increase the risk of mistakenly shooting a player.
```

Comment:
```suggestion
The main difficulties are that positions may quickly become outdated and also stealth may increase the risk of mistakenly shooting a teammate.
```

If they shoot any player, that's a bonus. A teammate is what you need to be careful about

Reply:
Sounds good, revising it a bit for better reading.

> The main difficulties are that positions may quickly become outdated and stealth may increase the risk of a player mistakenly shooting a teammate.